modules/virt-updating-virtio-drivers-windows.adoc

```diff
@@ -0,0 +1,27 @@
+// Module included in the following assemblies:
+//
+// * virt/virtual_machines/virt-installing-qemu-guest-agent.adoc
+// * virt/virtual_machines/virtual_disks/virt-managing-vm-snapshots.adoc
+
+:_content-type: PROCEDURE
+[id="virt-updating-virtio-drivers-windows_{context}"]
+= Updating VirtIO drivers on a Windows guest
+
+Update the `virtio` drivers by using Windows Update.
+
+.Prerequisites
+
+* A Windows guest operating system with the `virtio` drivers must be attached to the virtual machine (VM).
```

Comment:
This feature only works in connected environments.
In disconnected envs/clusters the windows update service can not be reached